AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Utils/IdentifyResultTransformers.swift

```diff
@@ -67,6 +67,96 @@ class IdentifyResultTransformers {
+            default: break
+            }
+        }
+        return keyText
+    }
+
+    static func processValue(ids: [String],
+                             blockMap: [String: AWSTextractBlock]) -> (String, Bool) {
+        var valueText = ""
+        var valueSelected = false
+
+        // VALUE block has a CHILD list of IDs for the WORD block
+        for valueId in ids {
+            let valueBlock = blockMap[valueId]
+            guard let valueRelatioins = valueBlock?.relationships else {
+                continue
+            }
+            for valueRelation in valueRelatioins {
+                guard let ids = valueRelation.ids else {
+                    break
+                }
+                for id in ids {
+                    let wordBlock = blockMap[id]
+                    guard let wordValueBlockType = wordBlock?.blockType else {
+                        continue
+                    }
+                    switch wordValueBlockType {
+                    case .word:
+                        if let text = wordBlock?.text {
+                            valueText += text + " "
+                        }
+                    case .selectionElement:
+                        valueSelected = wordBlock?.selectionStatus == .selected ? true : false
+                    default: break
+                    }
+                }
+            }
+
+        }
+        return (valueText, valueSelected)
+    }
+
+    static func parseLineBlock(block: AWSTextractBlock) -> IdentifiedLine? {
+        guard let text = block.text,
+            let boundingBox = processBoundingBox(block.geometry?.boundingBox),
+            let polygon = processPolygon(block.geometry?.polygon) else {
+                return nil
+        }
+
+        return IdentifiedLine(text: text,
+                              boundingBox: boundingBox,
+                              polygon: polygon,
+                              page: Int(truncating: block.page ?? 0))
+    }
+
+    static func parseWordBlock(block: AWSTextractBlock) -> IdentifiedWord? {
+        guard let text = block.text,
+            let boundingBox = processBoundingBox(block.geometry?.boundingBox),
+            let polygon = processPolygon(block.geometry?.polygon) else {
+                return nil
+        }
+
+         return IdentifiedWord(text: text,
+                               boundingBox: boundingBox,
+                               polygon: polygon,
+                               page: Int(truncating: block.page ?? 0))
+    }
+
+    static func parseSelectionElementBlock(block: AWSTextractBlock) -> Selection? {
+        guard let boundingBox = processBoundingBox(block.geometry?.boundingBox),
+            let polygon = processPolygon(block.geometry?.polygon) else {
+                return nil
+        }
+        let selectionStatus = block.selectionStatus == .selected ? true : false
+        return Selection(boundingBox: boundingBox, polygon: polygon, isSelected: selectionStatus)
+    }
+
```

Comment:
seems like we've added  the following functions to this class:
```
processChild
processValue
parseLineBlock
parseWordBlock
parseSelectionElementBlock
```

Why were they added to the parent level class, and not to IdentifyTextResultTransformers directly?

I can kinda understand why `parseLineBlock`, `parseWordBlock`, and `parseSelectionElementBlock` were added, but we'd probably want to rename these for consistency to `processLineBlock`, `processWordBlock`, and `processSelectionElementBlock` respectively... 

I'm not entirely convinced `processChild`, `processValue` belong in the parent class.

Reply:
The consistency of naming is a good point. But `Child` and `Value` are both `TextractBlock`, the same as `LineBlock`, `WordBlock` and `SelectionElementBlock`, so I put `processChildOfKeyValueSet()` and `processValueOfKeyValueSet()` in parent level